disputer/test/Disputer.js

```diff
@@ -202,4 +204,42 @@ contract("Disputer.js", function(accounts) {
     assert.equal((await emp.getLiquidations(sponsor2))[0].disputer, zeroAddress);
     assert.equal((await emp.getLiquidations(sponsor2))[0].state, STATES.DISPUTE_SUCCEEDED);
   });
+
+  it("Too little collateral", async function() {
+    // sponsor1 creates a position with 150 units of collateral, creating 100 synthetic tokens.
+    await emp.create({ rawValue: toWei("150") }, { rawValue: toWei("100") }, { from: sponsor1 });
+
+    // sponsor2 creates a position with 1.75 units of collateral, creating 1 synthetic tokens.
+    await emp.create({ rawValue: toWei("1.75") }, { rawValue: toWei("1") }, { from: sponsor2 });
+
+    // The liquidator creates a position to have synthetic tokens.
+    await emp.create({ rawValue: toWei("1000") }, { rawValue: toWei("500") }, { from: liquidator });
+
+    await emp.createLiquidation(
+      sponsor1,
+      { rawValue: toWei("1.75") },
+      { rawValue: toWei("100") },
+      { from: liquidator }
+    );
+
+    await emp.createLiquidation(sponsor2, { rawValue: toWei("1.75") }, { rawValue: toWei("1") }, { from: liquidator });
+
+    // Send most of the user's balance elsewhere leaving only enough to dispute sponsor1's position.
+    const transferAmount = (await collateralToken.balanceOf(disputeBot)).sub(toBN(toWei("1")));
+    await collateralToken.transfer(rando, transferAmount, { from: disputeBot });
+
+    // Both positions should be disputed with a presumed price of 1.1, but will only have enough collateral for the smaller one.
+    await disputer.queryAndDispute(time => toWei("1.1"));
+
+    // Only sponsor2 should be disputed.
+    assert.equal((await emp.getLiquidations(sponsor1))[0].state, STATES.PRE_DISPUTE);
```

Comment:
You should use the `LiquidationStates` now available in `common/Enums` here (can replace the object on line 33